carbonmark/package.json

```diff
@@ -5,7 +5,7 @@
     "dev": "next dev -p 3002",
     "build": "prettier --check . && npm run lingui:compile && next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "next lint && tsc --noEmit && prettier --check .",
```

Comment:
In a recent PR we moved `prettier` to a parallel GHA to speed up builds, so this is redundant

Reply:
Done